src/patternfly/components/Pagination/pagination.scss

```diff
@@ -102,7 +103,7 @@
   }
 
   .pf-c-form-control {
-    width: var(--pf-c-pagination__nav-page-select--c-form-control--Width);
+    width: calc(var(--pf-c-pagination__nav-page-select--c-form-control--Base) + (var(--pf-c-pagination__nav-page-select--c-form-control--Chars) * 1ch));
```

Comment:
And if you go with the code I suggested above, we can just revert this to what it was before. That way if someone has defined their own `var(--pf-c-pagination__nav-page-select--c-form-control--Width)`, it will still override PF's styles.